drivers/spi/Kconfig.sedi

```diff
@@ -0,0 +1,14 @@
+# Intel SEDI SPI driver configuration options
+#
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config SPI_SEDI
+	bool "SEDI SPI controller driver"
+	default y
+	depends on DT_HAS_INTEL_SEDI_SPI_ENABLED
+	help
```

Comment:
make similar as other SEDI drivers, see drivers/i2c/Kconfig.sedi and drivers/gpio/Kconfig.sedi

Reply:
will do